css/_vertical_filmstrip_overrides.scss

```diff
@@ -32,14 +46,7 @@
          * any parent is also fixed.
          */
         position: fixed;
-
-        /**
-         * z-index adjusting is needed because the video state indicator has to
-         * display over the filmstrip when no videos are displayed but still be
-         * clickable but its inline dialogs must display over the video state
-         * indicator when videos are displayed.
-         */
-        z-index: #{$tooltipsZ + 1};
```

Comment:
This bit of CSS was a remnant from when the video quality label (hd/sd) would need to move over remote videos in vertical filmstrip 1:1 mode. Since the local video has been move to the top of the screen, next to the video quality label, and didn't need to move over remote videos anymore, this bit of CSS was not doing much.